kitsune/messages/jinja2/messages/inbox.html

```diff
@@ -1,53 +1,60 @@
 {% extends "messages/base.html" %}
 {% from "messages/includes/macros.html" import avatar_link, name_link %}
 {% set title = _('Inbox') %}
-{% set crumbs = [(None, title)] %}
+{% set crumbs = [(url('messages.inbox'), _('Messages')),
+                 (None, title)] %}
 {% set active_tab = 'inbox' %}
 
 {% block content %}
-  <div class="sumo-page-section">
+<div class="sumo-page-section">
     <article id="inbox" class="main">
       <h1 class="sumo-page-heading">{{ title }}</h1>
       <div class="actions sumo-button-wrap">
         <a class="sumo-button primary-button" href="{{ url('messages.new') }}">{{ _('New Message') }}</a>
       </div>
-
-      {% if msgs %}
-        <form action="{{ url('messages.bulk_action') }}" method="POST">
-          <div class="sumo-button-wrap">
-            <input class="sumo-button" type="submit" name="mark_read" value="{{ _('Mark Selected as Read') }}">
-            <input class="sumo-button" type="submit" name="delete" value="{{ _('Delete Selected') }}">
-          </div>
-
-          <ol class="message-list">
-            {% for message in msgs.object_list %}
-              <li
```

Comment:
Why are you replacing the list with a set of divs? Isn't the structure that the list provides a better suited approach for an inbox? How about accessibility?

Reply:
Great catch on the accessibility shift - reverting to list.